contentcuration/contentcuration/frontend/channelEdit/vuex/assessmentItem/actions.js

```diff
@@ -3,18 +3,30 @@ import { isNodeComplete } from 'shared/utils/validation';
 import db from 'shared/data/db';
 import { TABLE_NAMES } from 'shared/data/constants';
 
-function updateNodeComplete(nodeId, context) {
-  const node = context.rootGetters['contentNode/getContentNode'](nodeId);
-  const complete = isNodeComplete({
-    nodeDetails: node,
-    assessmentItems: context.getters.getAssessmentItems(nodeId),
-    files: context.rootGetters['file/getContentNodeFiles'](nodeId),
-  });
-  return context.dispatch(
-    'contentNode/updateContentNode',
-    { id: nodeId, complete },
-    { root: true }
-  );
+function updateNodeComplete(nodeId, context, maxTries = 10, delayMs = 100) {
```

Comment:
Adding comments to explain the odd workaround we need here could help prevent this from regressions in the future